### PR TITLE
Notepad siddhantkhariwal

### LINKED ISSUE #27: Project : Notepad
**DESCRIPTION**:  Make a notepad website that saves notes in localStorage.

**GENERAL REQUIREMENTS:**

Create a website with good design and visuals.
Add delete button for deleting notes from local storage.

**USE CASE:**
Users can create notes and delete them.

**ADDITIONAL CHALLENGES :** 
Create a search button so that users can search notes.

### PR DESCRIPTION
**Pull request by siddhantkhariwal**
## Description

- Notepad app made with vanilla, HTML, CSS, and bootstrap.
Closes #27 

## Add GIF/ScreenShot 

- https://i.imgur.com/4GEkv2P.gifv

## Task Checklist
Put x in between the square brackets without any spaces to tick your Task.  Create a PR only after you complete all the tasks.
- [x ] You have created a new branch for contribution.
- [ x] Added README.md for your project in a specified folder.
- [ x] Add Demo in GIF or Screenshot form.
- [ x] You are Registered Officially for HacktoberFest 2021 as Partcipant.
- [ x] Followed Contribution Guidelines.

## Issue 
27

### CODE REVIEW

File: Notepad-siddhantkhariwal/Readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+*********Notepad***********
```
Comment (Harshil-Jani) [inline Notepad-siddhantkhariwal/Readme.md:1]: Update Readme as per contribution guidelines :
- Name of Project
- Demo of Project in .gif how your project is look 
- Description of project 
-  Requirements or Setup instructions
-  Name of Author(s)  
- [Demo Readme](https://github.com/Tenet-Coding/Hacktoberfest-Projects/blob/master/ConwaysGameOfLife/README.md)

**Review by Harshil-Jani — CHANGES_REQUESTED** [review]
Update README as per contribution guidelines.

**Comment by siddhantkhariwal** [comment]
> Update README as per contribution guidelines.

Hey, @Harshil-Jani I completed the readme file. please see through it.

**Review by soumik2012 — APPROVED** [review]
Nicely done @siddhantkhariwal

**Pull request merged**